doc/calculus-ted.xml

```diff
@@ -0,0 +1,299 @@
+    <!-- * @html                                                      -->
+    <!-- * @pdf may be the actual PDF or a page with download links   -->
+    <!-- * @print is a page with info, or link to a bookseller's page -->
+    <!-- * @ancillary1 is a workbook or WW problem sets or ...        -->
+    <!-- * @ancillary2 does not yet exist, send request               -->
+
+    <!-- N.B. These are used preferentially to create the most        -->
+    <!-- obvious link to your project.  All will (eventually) be      -->
+    <!-- available, but may not be as obvious.  The ordering is       -->
+    <!--                                                              -->
+    <!--   @landing > @html > @pdf > @print > no link                 -->
+    <!--                                                              -->
+    <!-- So, in particular, do not point @landing to your personal    -->
+    <!-- home page.  Only use it to point to a page that is           -->
+    <!-- *dedicated* to your project and will give a visitor a clear  -->
+    <!-- choice of paths from there.                                  -->
+
+<!--
+    <sites
+        landing="http://linear.pugetsound.edu/"
+        source="https://github.com/rbeezer/fcla"
+        html="http://linear.pugetsound.edu/fcla/"
+        pdf="http://linear.pugetsound.edu/download.html"
+        print="http://linear.pugetsound.edu/physical.html"
+        ancillary1=""
+        />
+-->
+    <!-- Description (Blurb) -->
+
+    <!-- N.B. Markup here (and elsewhere) is HTML, not PreTeXt!       -->
+    <!-- So use <i></i> for book titles and <a href=""></a> for       -->
+    <!-- hyperlinks.  (But note that the "sites" element is the       -->
+    <!-- best place to put links pointing to parts of  your project.  -->
+
+    <!-- onesentence (required) -->
+    <!--   * Likely visible, so short and succinct                    -->
+    <!--   * "description" (following) will amplify the "onesentence" -->
+    <!--   * HTML respected (e.g. <i> for italics), do not use an     -->
+    <!--     overall element (e.g. <p>) to surround entire sentence   -->
+
+    <!-- description (optional, but highly encouraged) -->
+    <!--   * optional                                                 -->
+    <!--   * a sequence of HTML "p" elements, lists should be OK      -->
+    <!--   * can contain simple markup, additional URLs               -->
+    <!--   * a narrative description of the project's aims            -->
+    <!--   * include distinctive features not possible above/below    -->
+    <!--   * knowled, so brevity not required                         -->
+    <!--   * HTML respected, do use elements like <p>                 -->
+
+    <onesentence>
+        These notes constitute a self-contained sequence for teaching, 
+	using an inquiry-based pedagogy, most of the traditional topics 
+	of Calculus I, II, & III in classes of up to 40 students. 
```

Comment:
As this is an XML file, it can't have a bare `&` character like this. You could change this to `&amp;`, but I might recommend just using the word 'and'.